### PR TITLE
Updates beagle-angular according to the new core structure

### PR DESCRIPTION
**Pull request by Tiagoperes**
**- What I did**
Adapts beagle-angular to work with the new structure of beagle-web-core.

**- How I did it**
Updated imports and some services usage.

**- How to verify it**
I ran this version of the lib for both Beagle Forms and Beagle Playground (every demo example + global context).

**- Description for the changelog**
Adapts beagle-angular to work with the new structure of beagle-web-core.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -16,11 +16,11 @@
 import { Type } from '@angular/core'
 import {
   BeagleConfig,
-  BeagleUIService,
+  BeagleService,
```
Comment (carolinegoncalveszup) [inline src/types.ts:19]: Esses dois imports podem ser um só ne?!

File: src/runtime/utils/beagle-map-keys-config.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@
 */
 
 import { DefaultSchema } from '@zup-it/beagle-web'
```
Comment (carolinegoncalveszup) [inline src/runtime/utils/beagle-map-keys-config.ts:17]: Acho que esses dois imports também poderiam ser um só

File: src/types.ts (hunk 1 above)
Comment (Tiagoperes) [reply src/types.ts:19]: feito!

File: src/runtime/utils/beagle-map-keys-config.ts (hunk 2 above)
Comment (Tiagoperes) [reply src/runtime/utils/beagle-map-keys-config.ts]: feito!

**Pull request merged**